app/src/main/res/values-fil/strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="action_search">Hanap</string>
+    <string name="app_name">Loyalty Card Wallet</string>
+    <string name="action_add">Add\tdagdag</string>
```

Comment:
This looks quite wrong too

Reply:
dagdag seems to be to increment from a quick web search. Deleted this translation for now and pinged the translator.